tools/Localizations/Sources/Localizations/ExportTask.swift

```diff
@@ -67,6 +61,30 @@ struct ExportTask {
         try! task.run()
         task.waitUntilExit()
     }
+    
+    // Get Locales that are in the Xcode project
+    private func getProjectLocales() -> (Array<String>) {
+    
+        var localesList:[String] = []
+
+        let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: xcodeProjPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path)
+        let filePaths = blockzillaFolder?.allObjects as! [String]
+        let textFilePaths = filePaths.filter{$0.contains(".lproj")}
+        for txt in textFilePaths {
+ 
+        if let index = txt.firstIndex(of: ".") {
+            let firstPart = txt.prefix(upTo: index)
+            localesList.append(String(firstPart))
+            }
+        }
+        var uniqueLocales = Array(Set(localesList))
```

Comment:
This is all fine - I tend to use NSString which has methids like filename.pathExtension and filename.deletingPathExtension ... 

String and NSString are interchangable I think, so you could do `blockzillaFolder?.allObjects as? [NSString]`

I'm a bit conflicted about this because `NS*` is old style but they have so many useful methods on them.